evals/skippy-cache-production-bench.py

```
@@ -729,10 +931,76 @@ def main() -> int:
+                with log_path.open("w") as log:
+                    cmd = [
+                        str(args.llama_server_bin),
+                        "--model", str(case.model_path),
+                        "--ctx-size", str(max(case.ctx_size, case.prefix_tokens + 128)),
+                        "--n-gpu-layers", str(case.n_gpu_layers),
+                        "--host", "127.0.0.1",
+                        "--port", str(port),
+                        "--parallel", str(args.llama_parallel),
+                        "--no-webui",
+                    ]
+                    proc = subprocess.Popen(cmd, cwd=REPO, text=True, stdout=log, stderr=subprocess.STDOUT)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)